statsd/install_statsd.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -ex
+rm -rf ~/build_statsd
```

Comment:
Ahh... I didn't realize this was being done as root. I was thrown off by the sudo command. :+1: